Civi/Angular/Manager.php

```diff
@@ -289,13 +283,10 @@ public function getRawPartials($name) {
    *   Invalid partials configuration.
    */
   public function getPartials($name) {
-    $cacheKey = "angular-partials_$name";
-    $cacheValue = $this->cache->get($cacheKey);
-    if ($cacheValue === NULL) {
-      $cacheValue = ChangeSet::applyResourceFilters($this->getChangeSets(), 'partials', $this->getRawPartials($name));
-      $this->cache->set($cacheKey, $cacheValue);
+    if (!isset($this->partials[$name])) {
+      $this->partials[$name] = ChangeSet::applyResourceFilters($this->getChangeSets(), 'partials', $this->getRawPartials($name));
```

Comment:
OK, cool. So it looks like it would still be reading live HTML files. It's just using `$this->partials`(`array`)  instead of `CRM_Utils_Cache_ArrayCache`.